src/test/java/api/loans/CheckInByBarcodeTests.java

```diff
@@ -1369,6 +1369,80 @@ void overdueFineCalculatedCorrectlyWhenHourlyFeeFinePolicyIsApplied() {
       homeLocation.getJson().getString("name"), ownerId, feeFineId, 7.0));
   }
 
+  @Test
+  void overdueRecallFineCalculatedCorrectlyWhenRecallRequestCreated() {
+    double maxOverdueFine = 5.0;
+    double maxOverdueRecallFine = 30.0;
+    double overdueRecallFine = 6.0;
+    configurationsFixture.enableTlrFeature();
+    useFallbackPolicies(
+      loanPoliciesFixture.create(new LoanPolicyBuilder()
+        .withId(UUID.randomUUID())
+        .withName("1 minute policy")
+        .withDescription("Can circulate item")
+        .rolling(Period.minutes(1))
+        .withGracePeriod(Period.zeroDurationPeriod())
+        .unlimitedRenewals()
+        .renewFromSystemDate()).getId(),
+      requestPoliciesFixture.allowAllRequestPolicy().getId(),
+      noticePoliciesFixture.activeNotice().getId(),
+      overdueFinePoliciesFixture.create(new OverdueFinePolicyBuilder()
+        .withId(UUID.randomUUID())
+        .withName("One per minute overdue fine and overdue recall fine policy")
+        .withCountClosed(true)
+        .withGracePeriodRecall(false)
+        .withOverdueFine(
+          new JsonObject()
+            .put("quantity", 1.0)
+            .put("intervalId", "minute"))
+        .withMaxOverdueFine(maxOverdueFine)
+        .withOverdueRecallFine(new JsonObject()
+          .put("quantity", overdueRecallFine)
+          .put("intervalId", "minute"))
+        .withMaxOverdueRecallFine(maxOverdueRecallFine)
+        .withCountClosed(false)).getId(),
+      lostItemFeePoliciesFixture.facultyStandard().getId()
+    );
+    final IndividualResource homeLocation = locationsFixture.mainFloor();
+    final IndividualResource nod = itemsFixture.basedUponNod(item ->
+      item.withPermanentLocation(homeLocation.getId()));
+
+    ZonedDateTime checkOutDate = ZonedDateTime.of(2020, 1, 18, 18, 0, 0, 0, UTC);
+    ZonedDateTime requestDate = ZonedDateTime.of(2020, 1, 19, 18, 0, 0, 0, UTC);
+    ZonedDateTime checkInDate = ZonedDateTime.of(2020, 1, 22, 15, 30, 0, 0, UTC);
```

Comment:
How about making the checkin date in minutes after the request date, so that we can see that the amount of overdue recall fees is really different?